examples/README.md

```diff
@@ -1,4 +1,18 @@
 # Configuration examples
 
-- [Basic setup on Organization namespaces](https://github.com/giantswarm/linkerd-bundle/blob/main/examples/basic-org-ns.yaml)
+The following examples can be used to get started quickly. Please review the files thoroughly and adapt things like configuration and cluster names.
+
+A successful install will require you to generate a trust anchor and issuer certificate. The following steps loosely follow [the official instructions](https://linkerd.io/2.12/tasks/generate-certificates/).
+
+Obtain the `step` cli (we're using `step_linux_0.19.0_amd64.tar.gz` from [here](https://github.com/smallstep/cli/releases/tag/v0.19.0)) and execute the following commands. Take note of the `--not-after` flag. (8760h = 1 year)
+
+```bash
+step certificate create root.linkerd.cluster.local ca.crt ca.key --profile root-ca --no-password --insecure --not-after=8760h
+step certificate create identity.linkerd.cluster.local issuer.crt issuer.key --profile intermediate-ca --not-after=8760h --no-password --insecure --ca ca.crt --ca-key ca.key
+```
```

Comment:
We should probably also mention the PSP patch that is needed for linkerd to work. it can be done after deployment though:

From https://github.com/giantswarm/linkerd-control-plane-app/blob/main/README.md#step-2-deploy-linkerd

> Please note, that for security reasons Giant Swarm by default forbids the usage of emptyDir volumes as  storage for pods. Linkerd needs this functionality to run the proxy containers in any deployment that is going to be included in the service mesh. Enabling emptyDir volumes poses a risk that a pod will create such big emptyDir that the underlying cluster node will run out of disk space. If you're OK with this potential issue, the easiest way to allow for emptyDir for all the deployments is to edit the default PSP of your cluster by running the command:
>
> `kubectl patch psp restricted --type='json' -p='[{"op": "add", "path": "/spec/volumes/-", "value": "emptyDir"}]'`

Reply:
yes makes sense. Included :+1: